src/command/backup/backup.c

```diff
@@ -2253,6 +2253,10 @@ cmdBackup(void)
                     (BackupType)cfgOptionStrId(cfgOptType), manifestData(manifest)->backupLabelPrior, timestampStart));
         }
 
+        // Set annotations in the manifest
+        if (cfgOptionTest(cfgOptAnnotation))
+            manifestAnnotationSet(manifest, cfgOptionKv(cfgOptAnnotation));
+
```

Comment:
There does not seem to be any reason for this to be a separate function. I think annotations should be handled by `manifestBuildComplete()` (like `dbList`).

Reply:
This should be fixed in cc80268

Reply:
Looks much simpler now.